telegram/_chatjoinrequest.py

```diff
@@ -39,17 +39,32 @@ class ChatJoinRequest(TelegramObject):
     considered equal, if their :attr:`chat`, :attr:`from_user` and :attr:`date` are equal.
 
     Note:
-        Since Bot API 5.5, bots are allowed to contact users who sent a join request to a chat
-        where the bot is an administrator with the
-        :attr:`~telegram.ChatMemberAdministrator.can_invite_users` administrator right – even if
-        the user never interacted with the bot before.
+        * Since Bot API 5.5, bots are allowed to contact users who sent a join request to a chat
+            where the bot is an administrator with the
+            :attr:`~telegram.ChatMemberAdministrator.can_invite_users` administrator right – even
+            if the user never interacted with the bot before.
+        * Telegram does not guarantee that :paramref:`from_user` ``.id`` coincides with the
+            ``chat_id`` of the user. Please use :paramref:`user_chat_id` to contact the user in
+            response to their join request.
```

Comment:
```suggestion
        * Since Bot API 5.5, bots are allowed to contact users who sent a join request to a chat
          where the bot is an administrator with the
          :attr:`~telegram.ChatMemberAdministrator.can_invite_users` administrator right – even
            if the user never interacted with the bot before.
        * Telegram does not guarantee that :attr:`from_user.id <from_user>` coincides with the
          ``chat_id`` of the user. Please use :attr:`user_chat_id` to contact the user in
          response to their join request.
```
fix rendering, paramref -> attr